docs/ingressdns-with-externaldns.md

```diff
@@ -0,0 +1,237 @@
+# Federated Ingress DNS with ExternalDNS Guide
+
+Federated Ingress DNS (FIDNS) provides the ability to programmatically manage DNS records of Ingress objects.
+FIDNS is not meant to replace in-cluster DNS providers such as [CoreDNS](https://coredns.io/). Instead, FIDNS works with
+systems such as [ExternalDNS](https://github.com/kubernetes-incubator/external-dns) that are responsible for managing
```

Comment:
I would rephrase this as "FIDNS leverages ExternalDNS to enable reprogramming many different DNS providers", as is it sounds like external-dns is one of many things that can be used, when it's the only way :)